Solutions/Marain.TenantManagement.Specs/Data/ServiceManifests/OperationsServiceManifest.json

```diff
@@ -1,8 +1,11 @@
 {
   "$schema": "./ServiceManifestSchema.json",
   "contentType": "application/vnd.marain.tenancy.servicemanifests.servicemanifest",
+  "wellKnownTenantGuid": "4f522924-b6e7-48cc-a265-a307407ec858", // 3633754ac4c9be44b55bfe791b1780f12429524fe7b6cc48a265a307407ec858
```

Comment:
Another post-mangling ID in the comments.

Reply:
Is the mangling worth the candle? Could we just unmangle and let you see the path?

Reply:
Downsides: it is less URI friendly